corehq/apps/app_manager/views/modules.py

```diff
@@ -770,6 +776,9 @@ def should_edit(attribute):
         except InvalidSessionEndpoint as e:
             return HttpResponseBadRequest(str(e))
 
+    if should_edit('custom_assertions'):
+        handle_custom_assertions(request, module, lang)
```

Comment:
note that I'm not handling the possible error response here - that's because I plan to catch it in a different way in the future.  Stay tuned.